chart/compass/values.yaml

```diff
@@ -79,9 +79,20 @@ global:
     caKey: ""
     caCertificate: ""
 
+  kyma_environment_broker:
+    secrets:
+      integrationSystemCredentials:
+        name: compass-kyma-environment-broker-credentials
+
   provisioner:
     graphql:
       port: 3000
+    secrets:
+      integrationSystemCredentials:
+        name: compass-provisioner-credentials
+
+  oauth2:
+    host: oauth2.kyma.local
```

Comment:
Maybe we can wrap it under `local` like this:
```
global
  localEnv:
    oauth2:
        host: ...
```
To signify it is used only on local environment. Or preferably put only a first part of host like:
```
global
  oauth2:
      host: oauth2
```
and use `.{{ .Values.global.ingress.domainName }}` to complete the full domain.